src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,153 @@
 package core.basesyntax;
+//import jdk.internal.util.ArraysSupport;
+
+import java.util.Arrays;
+import java.util.NoSuchElementException;
 
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_CAPACITY = 10;
+    private static final Object[] EMPTY_ARRAYDATA = {};
+    private T[] arrayData = (T[]) new Object[DEFAULT_CAPACITY];
+    private int size;
+    private int cursor;
```

Comment:
you don't need this global variable, remove it

Reply:
and remove the global variable "size"?

Reply:
and array?